User Guide 3.2/Tests/tests-settings.md

```diff
@@ -36,4 +48,6 @@ The *Selection* panel asks if the delivered test section should include only som
 
 This brings up three panels: general properties, *Item Session Control*, and *Time Limits*. The last two panels are the same as the last two in both the test part and section level properties. The general properties include entries for: *Identifier*, *Reference*, and *Categories*. It also includes check boxes to indicate the item is *Required* and if it is *Fixed*.
 
+![Settings for Test Items](../resources/backend/tests/authoring/settings/item/properties.png)
+
 The *Identifier* and *References* boxes generally do not require modification. *Categories* act as tag references which may be displayed to the test-taker. If the *Required* box is checked, the Item will appear on the Test, even if less than the total number of Items appears on a given Test (in a section where selection is enabled). If the *Fixed* box is checked, the Item will appear in a particular order, even if the section ordering calls for shuffling.
```

Comment:
Categories are not displayed to the test-taker, they are typically used to calculate aggregate scores (e.g. on sub-domains), see scoring rules.